packages/orbit-components/src/Seat/Seat.stories.tsx

```diff
@@ -66,17 +66,18 @@ export const Playground: Story = {
   },
 };
 
-export const Legend: Story = {
+export const Legend: StoryObj<typeof SeatLegend> = {
   render: args => <SeatLegend {...args} />,
 
   parameters: {
     info: "SeatLegend component. Check Orbit.Kiwi for more detailed guidelines.",
     controls: {
-      exclude: ["size", "title", "description", "selected", "onClick"],
+      exclude: ["size", "title", "description", "selected", "onClick", "aria-labelledby"],
```

Comment:
Hmm no, because if I do that the `title` and `description` will be visible, even though they're unrelated. I have them defined on the main story, so I need to exclude them here.